### DISCUSSION TITLE
Enable IPv6 on mx1 and mx2

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1703] [Feature Requests]
Hi!

Would it be possible for you to implement and enable IPv6 on:

mx1.simplelogin.co
mx2.simplelogin.co

Thank you!

**Comment by SteffenNS** [comment]
A move from IPv4 to IPv6 is desirable for freedom lovers because IPv6 has enough IP addresses to enable true peer-to-peer communication between internet users. With the limited number of IPv4 addresses, most IPv4 users are behind NAT equipment, and can really only create outbound traffic, not accept inbound traffic. Therefore we currently need our traffic to go through tech giants like Facebook, Twitter, Google etc.
